src/model/transaction/__tests__/__snapshots__/getContentStateFragment-test.js.snap

```diff
@@ -261,7 +261,7 @@ Object {
     "data": Object {},
     "depth": 0,
     "key": "key10",
-    "nextSibling": "D",
+    "nextSibling": null,
```

Comment:
This was a bug that I managed to notice while developing this PR and fixing the randomizeBlockMapKeys.